controls/stig_rhel9.yml

```diff
@@ -2315,7 +2315,6 @@ controls:
         title: RHEL 9 must have the USBGuard package installed.
         rules:
             - package_usbguard_installed
-            - service_usbguard_enabled
```

Comment:
This probably shouldn't be here

Reply:
It is covered by [RHEL-09-291020](https://stigaview.com/products/rhel9/v2r4/RHEL-09-291020). So that isn't needed. Also see [RHEL-09-291015](https://stigaview.com/products/rhel9/v2r4/RHEL-09-291015)